go/test/endtoend/vreplication/vreplication_test_env.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package vreplication
 
 var dryRunResultsSwitchWritesCustomerShard = []string{
+	"Mirroring 0.000000 percent of traffic from keyspace product to keyspace customer for tablet types [PRIMARY]",
```

Comment:
Is the precision of the percent value intentional? 6 digits of precision seems unnecessary. It will be more user-friendly to limit this to either an integer or at most 2 digits of precision.

Reply:
not intentional, good call

Reply:
done in [71b9a00](https://github.com/vitessio/vitess/pull/15945/commits/71b9a008952a81fa325249efdf5658a92991c92d)